apps/site/pages/components/atoms/input.mdx

```diff
@@ -4,12 +4,40 @@ sidebar_custom_props:
   image: https://vtexhelp.vtexassets.com/assets/docs/src/Input___77d0e7a8079b7d310e8223dd9e1994f6.png
```

Comment:
Can you do me a big favor pls? 🌷 

I've noticed that we made a mistake on those examples. 
Can you change this for me pls?

It should be: (We need to change in the `Example` and also in the `Code` tab.

```
<Input placeholder="Label" id="inputfield-default" />
<Input placeholder="Input Disabled" id="inputfield-disabled" disabled />
```

<img width="893" alt="image" src="https://github.com/vtex/faststore/assets/3356699/e3ede573-4839-4fa3-acea-cd20bf213307">